packages/lib/formulas/logarithm/handler.test.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, test } from '@jest/globals'
+import handler from './handler'
+
+describe('Formula: Logarithm', () => {
+  test('should return the logarithm of the number', () => {
+    expect(handler([-1], undefined as any)).toBe(null)
+    expect(handler([0], undefined as any)).toBe(-Infinity)
+    expect(handler([1], undefined as any)).toBe(0)
+    expect(handler([10], undefined as any)).toBe(2.302585092994046)
```

Comment:
It would be great if you could a few more test cases for `null`, `undefined` and a `string` perhaps:
```suggestion
    expect(handler([10], undefined as any)).toBe(2.302585092994046)
    expect(handler([null], undefined as any)).toBe(null)
    expect(handler([], undefined as any)).toBe(null)
    expect(handler(["hello"], undefined as any)).toBe(null)
```

Reply:
One day I'll be as good as the god of code